test/integration/Highlight.e2e.test.js

```diff
@@ -0,0 +1,30 @@
+// <reference types="Cypress" />
```

Comment:
Are these tags still needed?

Reply:
Will convert to triple slashes in a separate PR.